crates/fadroma/scrt/snip20/contract/snip20.rs

```diff
@@ -339,13 +340,13 @@ pub(crate) mod default_impl {
                     amount = coin.amount
                 } else {
                     return Err(StdError::generic_err(
-                        "Tried to deposit an unsupported token",
+                        "Tried to deposit an unsupported token.",
```

Comment:
How about we unify all of the error messages so they match with the reference implementation :thinking: ?

I doubt anyone is matching the raw string, but the less difference the better IMO.

Reply:
Seems like a big rabbit hole to go into without a discernible benefit. Most error messages are already identical. Some are not because of difference in implementation or because I've decided on a better alternative or simply formatted them like this one. Also, we never know when they will go out of sync and they are not a part of the
SNIP-20 standard anyways. What is part of the standard is the response data that each message includes which we have.